apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -584,6 +584,22 @@ type AdmissionControllerFeatureConfig struct {
 	// Default: "datadog-webhook"
 	// +optional
 	WebhookName *string `json:"webhookName,omitempty"`
+
+	// CWSInstrumentation holds the CWS Instrumentation endpoint configuration
+	// +optional
+	CWSInstrumentation *CWSInstrumentationFeatureConfig `json:"cwsInstrumentation,omitempty"`
```

Comment:
👋🏻 I don't have any strong opinion on this, I've renamed `CWSInstrumentationFeatureConfig` to `CWSInstrumentationConfig` for now 👍🏻 

Personally speaking I like the CWS nesting because it mimics the agent configuration schema and it isolate the CWS related parameters together. But I'm happy to change if you want me too.